pkg/minikube/config/extra_options.go

```diff
@@ -65,6 +65,14 @@ func (es *ExtraOptionSlice) Exists(value string) bool {
 
 // Set parses the string value into a slice
 func (es *ExtraOptionSlice) Set(value string) error {
+
+	// Check we don't end with suffix quotation.
+	prefixExists := strings.HasPrefix(value, "”") || strings.HasPrefix(value, "“")
+	suffixExists := strings.HasSuffix(value, "”") || strings.HasSuffix(value, "“")
+	if !prefixExists && suffixExists {
+		return fmt.Errorf("invalid value: cannot contain end quotation: %q", value)
```

Comment:
what happens when error happens ? could u paste in the PR description

Before and After this PR ? 
how about adding Unit test for `Set` Function ? with example of things that should Error and example of things that should NOT error

Reply:
also improve the error wording, ```-extra-config```

Reply:
I've added a test and in the PR description above I've included what happens when we do the error checking. We are stopping he hanging occurring when a extra-config value ends with a open or close quote. It's important to note if the extra-config value has a prefix with an open or close quote then there's a separate error check in start.go file (cli) - I have left that alone.